index.js

```diff
@@ -1,10 +1,16 @@
 import {$if, $with, ifnot} from './src/ifIfnotWith'
-import {template} from './src/template'
+import {foreach} from './src/foreach'
+import {template} from './src/templating'
 
 export var bindings = {
     'if': $if,
     'with': $with,
     ifnot: ifnot, unless: ifnot,
-    'let': letBinding,
-    template: template,
+    foreach: foreach,
```

Comment:
Relevant note:  I am hoping to switch the default foreach binding provider to https://github.com/brianmhunt/knockout-fast-foreach (which will become tko.binding.foreach, I guess).  The old binding behaviour will still be available with `template: { foreach: $data }`.